content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,352 @@
-000003 and 000002.
+Then, :guilabel:`Create` a sales order for four white shirts and :guilabel:`Confirm` it. Check that
+the reserved products are from lots 000003 and 000002 by looking at the :guilabel:`Detailed
+Operations` in the  :guilabel:`Sales Order`.
 
-.. image:: removal/reserved-lots-LIFO.png
+.. image:: removal/reserved-lots-lifo.png
    :align: center
-   :alt: two lots being reserved for sell with the LIFO strategy.
-
-.. important::
-   Don’t forget that the *LIFO* strategy is banned in many countries!
+   :alt: Two lots being reserved for sale with the LIFO strategy.
 
-FEFO (First Expiry, First Out)
-------------------------------
+FEFO (First Expired, First Out)
+-------------------------------
 
-With the *FEFO* strategy, the way products are picked is not based on the reception date. In this
-particular case, they are dispatched according to their expiration date.
+As explained earlier, the :abbr:`FEFO (First Expiry, First Out)` removal strategy differs from the
+:abbr:`FIFO (First In, First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it
+targets products for removal based on **expiration dates** instead of their warehouse receipt dates.
 
 .. note::
-   To have more information about Expiration date, please have a look at
+   For more information about expiration dates, reference
    :doc:`the related doc <../../management/lots_serial_numbers/expiration_dates>`.
 
-By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot
-numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory -->
-Master Data --> Lots/Serial Numbers`.
+Once the expiration dates feature is activated, it's possible to define different expiration dates
+for individual serialized products, **or** lot numbers containing many products.
+
+Expiration dates can be set by going to :menuselection:`Inventory app --> Products --> Lots/Serial
+Numbers`, or they can be entered when validating the received products.
 
 .. image:: removal/removal-date.png
    :align: center
-   :alt: view of the removal date for 0000001.
+   :alt: View of the removal date for 0000001.
 
 Lots are picked based on their removal date, from earliest to latest. Lots without a removal date
 defined are picked after lots with removal dates.
 
-.. note::
-   Other dates are for informational and reporting purposes only. If not removed from the stock,
-   lots that are past the expiration dates may still be picked for delivery orders!
-
-To use the *FEFO* strategy, once again go to :menuselection:`Inventory --> Configuration -->
-Product Categories` and choose *FEFO* as the *Force Removal Strategy*.
-
-.. image:: removal/first-expiry-first-out.png
-   :align: center
-   :alt: view of the FEFO strategy being set up as forced removal strategy.
-
-For this particular case, let’s use hand cream. As usual, we have three lots of them.
+.. important::
```

Comment:
```suggestion
.. warning::
```
 I would argue this gets a warning tag, since sending expired products to customers could be pretty serious

Reply:
very good idea- I agree!